File: Assignment6/program1.c
//Write a program which accepts name from the user and print that name

#include<stdio.h>

int main()
{
    char Name[30];
    printf("Please Enter Full Name: \n");
    scanf("%s", &Name);
    printf("Your name is %s",  Name);
    return 0;
}
//Time Complexity: O(1)

File: Assignment56/program1.java
//1. Write a java program to accept file name from user and open that file.
import java.io.File;
import java.io.IOException;
import java.util.Scanner;
class program1
{
    public static void main(String A[])
    {
        try 
        {
            Scanner sobj = new Scanner(System.in);

            String FileName;
            System.out.print("Enter the name if the file : ");
            FileName = sobj.next();

            File f = new File(FileName);

                if (f.createNewFile()) 
                {
                    System.out.println("File created successfully");
                } else
                {
                    System.out.println("File already exists");
                }
        } 
        catch (IOException e) 
        {
            System.out.println("Error creating file");
        }
    }
}
